Dockerfile

```diff
@@ -123,18 +59,33 @@ RUN if [ ! -f /tmp/resources/jai-1_1_3-lib-linux-amd64.tar.gz ]; then \
     rm /tmp/jai-1_1_3-lib-linux-amd64.tar.gz && \
     rm -r /tmp/jai-1_1_3 && \
     rm /tmp/jai_imageio-1_1-lib-linux-amd64.tar.gz && \
-    rm -r /tmp/jai_imageio-1_1
+    rm -r /tmp/jai_imageio-1_1; \
+    fi
+
+# Add GDAL native libraries if the build-arg GDAL_NATIVE = true
+ARG GDAL_NATIVE=true
+# EWC and JP2ECW are subjected to licence restrictions
+
+ENV GDAL_DATA $CATALINA_HOME/gdal-data
+ENV LD_LIBRARY_PATH $JAVA_HOME/jre/lib/amd64/gdal:/usr/local/lib:/usr/lib/x86_64-linux-gnu:/opt/libjpeg-turbo/lib64
+RUN if [ "$GDAL_NATIVE" = true ]; then \
+    wget http://demo.geo-solutions.it/share/github/imageio-ext/releases/1.1.X/1.1.12/native/gdal/gdal-data.zip && \
+    wget http://demo.geo-solutions.it/share/github/imageio-ext/releases/1.1.X/1.1.12/native/gdal/linux/gdal192-Ubuntu12-gcc4.6.3-x86_64.tar.gz && \
+    unzip gdal-data.zip -d $CATALINA_HOME && \
+    mkdir $JAVA_HOME/jre/lib/amd64/gdal && \
+    tar -xvf gdal192-Ubuntu12-gcc4.6.3-x86_64.tar.gz -C $JAVA_HOME/jre/lib/amd64/gdal; \
+    fi
```

Comment:
Gdal native should be installed by default. This allows a user to use the plugin without needing to make other configurations